src/scripts/domParsers/domUtils.ts

```diff
@@ -195,6 +195,17 @@ export class DomUtils {
 		return sanitizedHtml;
 	}
 
+	/**
+	 * Many extensions inject their own stylings into the page, and generally that isn't a problem. But,
+	 * occasionally the styling includes a specific font, which can be very, very large. This method
+	 * removes any base64 encoded binaries defined in any <style> tags.
+	 */
+	public static removeStylesWithBase64EncodedBinaries(doc: Document): void {
+		DomUtils.domReplacer(doc, "style", (node: HTMLElement) => {
+			return node.innerHTML.indexOf("data:application") !== -1 ? document.createElement("style") : node;
```

Comment:
We should be able to return undefined instead of an empty style element to cut down on the HTML we send over the wire.